src/server/packageJsonCache.ts

```diff
@@ -0,0 +1,56 @@
+/*@internal*/
+namespace ts.server {
+    export interface PackageJsonCache {
+        addOrUpdate(fileName: Path): void;
+        delete(fileName: Path): void;
+        getInDirectory(directory: Path): PackageJsonInfo | undefined;
+        entries(): Iterator<[Path, PackageJsonInfo]>;

```

Comment:
Where is `entries()` used? I couldnt find it.. If its, i would recommend removing it.. If its for debug pupose, print or getData which returns a string is better.. Esp since it returns Path which can be used as string and i dont want people confusing Path which might not be same as fileName